Dockerfile

```diff
@@ -34,9 +34,17 @@ RUN bash scripts/install_onnx_gpu_for_amd.sh && \
     bash scripts/install_torch_amd.sh && \
     bash scripts/install_decord.sh && \
     bash scripts/install_redis.sh && \
-    bash scripts/install_ffmpeg.sh && \
     bash scripts/install_punkt_tokenizers.sh
 
+# Install ffmpeg based on the architecture
+RUN if [ "${TARGETPLATFORM}" = "linux/arm64" ]; then \
```

Comment:
Can we do staged build for cuda ffmpeg? use the same base image and build from source, then copy only the binaries over.